layouts/shortcodes/eol-releases.html

```diff
@@ -0,0 +1,34 @@
+<h2 id="non-active-branch-history">Non-Active Branch history</h2>
+<p>
+    These releases are no longer supported.
+</p>
+<div>
+    <table class="table table-striped col-md-8">
```

Comment:
nice-to-have: set a class on this table that indicates it's a table of EOL releases.